From 6a85cf9b89737284e3839c86760e3379bf77ed91 Mon Sep 17 00:00:00 2001
From: Sylwester Nawrocki <s.nawrocki@samsung.com>
Date: Fri, 8 Feb 2013 19:26:33 +0100
Subject: [PATCH 0060/1302] regulator: max77693: Remove no longer supported
 __dev* attributes
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This patch fixes following build error:

drivers/regulator/max77693.c:673: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘int’
drivers/regulator/max77693.c:743: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘max77693_pmic_remove’
drivers/regulator/max77693.c:768: error: ‘max77693_pmic_probe’ undeclared here (not in a function)
drivers/regulator/max77693.c:769: error: implicit declaration of function ‘__devexit_p’
drivers/regulator/max77693.c:769: error: ‘max77693_pmic_remove’ undeclared here (not in a function)

Signed-off-by: Sylwester Nawrocki <s.nawrocki@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/regulator/max77693.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/regulator/max77693.c b/drivers/regulator/max77693.c
index 85776b2..0720ea8 100644
--- a/drivers/regulator/max77693.c
+++ b/drivers/regulator/max77693.c
@@ -670,7 +670,7 @@ static struct regulator_desc regulators[] = {
 	},
 };
 
-static __devinit int max77693_pmic_probe(struct platform_device *pdev)
+static int max77693_pmic_probe(struct platform_device *pdev)
 {
 	struct max77693_dev *iodev = dev_get_drvdata(pdev->dev.parent);
 	struct max77693_platform_data *pdata = dev_get_platdata(iodev->dev);
@@ -740,7 +740,7 @@ static __devinit int max77693_pmic_probe(struct platform_device *pdev)
 	return ret;
 }
 
-static int __devexit max77693_pmic_remove(struct platform_device *pdev)
+static int max77693_pmic_remove(struct platform_device *pdev)
 {
 	struct max77693_data *max77693 = platform_get_drvdata(pdev);
 	struct regulator_dev **rdev = max77693->rdev;
@@ -766,7 +766,7 @@ static struct platform_driver max77693_pmic_driver = {
 		   .owner = THIS_MODULE,
 		   },
 	.probe = max77693_pmic_probe,
-	.remove = __devexit_p(max77693_pmic_remove),
+	.remove = max77693_pmic_remove,
 	.id_table = max77693_pmic_id,
 };
 
-- 
1.8.3.2

